src/js/components/TextArea/stories/NonResizable.js

```diff
@@ -11,7 +11,13 @@ const Resize = (props) => {
     // Uncomment <Grommet> lines when using outside of storybook
     // <Grommet theme={...}>
     <Box align="center" pad="large">
-      <TextArea value={value} onChange={onChange} {...props} />
+      <TextArea
+        label="text area"
```

Comment:
```suggestion
```
I think all we need here is an `aria-label` and we can remove the extra `label` here